babel-build.config.js

```diff
@@ -0,0 +1,16 @@
+const rootBabel = require('./babel.config');
+
+// TODO: Remove this once https://github.com/storybookjs/storybook/issues/11246 is fixed
+const popper2AliasRemovalPlugin = [
+    'babel-plugin-module-resolver',
+    {
+        alias: {
+            'react-popper-2': 'react-popper'
```

Comment:
I'm jumping through hoops to get the generated output to look like
```js
var _reactPopper = require("react-popper");
```
instead of
```js
var _reactPopper = require("react-popper-2");
```
and still have the examples work in storybook